paf-mvp-core-js/tests/test-cases/node.test.ts

```diff
@@ -0,0 +1,196 @@
+import { IJsonValidator, JsonSchemaType, JsonValidation } from '@core/validation/json-validator';
+import { IdentityConfig, Node } from '@core/express';
+import { createRequest, createResponse, MockResponse } from 'node-mocks-http';
+import { NextFunction, Response } from 'express';
+import { NodeError, NodeErrorType } from '@core/errors';
+import { getTimeStampInSec } from '@core/timestamp';
+import { encodeBase64, QSParam } from '@core/query-string';
+
+const buildStaticJsonValidator = (validationResult: boolean): IJsonValidator => {
```

Comment:
Does it make sense to create mocks in separate file so that it can be reused and simplify this file.
```typescript
class JsonValidatorMock implements IJsonValidator {

    constructor(public returnError: boolean) {}

    start() {
        return Promise.resolve();
    }

    validate(schema: JsonSchemaType, jsonStr: string): JsonValidation {

        if (this.returnError) {
            return {
                isValid: false,
                errors: [new Error('error message from validator')],
              };
        } else {
            return {
                isValid: true,
                value: {},
              };
        }
    }
}
```